argoverse/evaluation/eval_tracking.py

```diff
@@ -6,15 +6,17 @@
 import pathlib
 import pickle
 from pathlib import Path
-from typing import Any, Dict, List, TextIO, Union
+from typing import Any, Dict, List, Optional, TextIO, Union
 
 import motmetrics as mm
 import numpy as np
 from shapely.geometry.polygon import Polygon
 
 from argoverse.evaluation.eval_utils import label_to_bbox
 from argoverse.utils.json_utils import read_json_file
-from argoverse.utils.se3 import SE3
+from click.core import Option
```

Comment:
i dont think we have `click` as a dependency, do we? could we get `Option`-like functionality from another library?

Reply:
not sure how that got added. removed.